Content.Client/GameObjects/EntitySystems/ExamineSystem.cs

```diff
@@ -141,6 +173,28 @@ public async void DoExamine(IEntity entity)
                     break;
                 }
             }
+
+            var size = Vector2.ComponentMax((minWidth, 0), panel.CombinedMinimumSize);
+            size = Vector2.ComponentMax(size, _examineTooltipOpen.CombinedPixelMinimumSize);
```

Comment:
Here I then check again for the more realistic size of the popup now that all the children have been added. Later on I can use the actual size to see how much I need to push the popup back into the screen so that when we open it see all of the popup